v-next/hardhat-ignition/src/helpers.ts

```diff
@@ -0,0 +1,5 @@
+export { HardhatArtifactResolver } from "./helpers/hardhat-artifact-resolver.js";
+export { errorDeploymentResultToExceptionMessage } from "./helpers/error-deployment-result-to-exception-message.js";
```

Comment:
What types should we export to match this?
Do we re-export core types that are in these helper function's signatures?

Reply:
not sure i understand this

Reply:
We have a packages export for `helpers` (they are used from the companion plugins).
In `hardhat-ignition-core` we use the API extractor to ensure that all exported types are available.
Should we be doing the same for the helpers here?

Reply:
We will revisit the type exports after the initial public alpha